src/scene.py

```diff
@@ -0,0 +1,65 @@
+import random
+from src.common.point import Point
+from src.objects.agent import Agent
+from src.objects.floor import Floor
+from src.objects.portal import Portal
+from .objects.object import Object
+
+
+class Scene():
+    '''
+    Object representing one level in the game.
+    It is responsible for storing all instances of `Object` class.
+    '''
+
+    def __init__(self, runtime) -> None:
+        self.objects_count = 0
+        self.objects_map = {}  # Point : [Object] Field
```

Comment:
Instead of using comments, we can just declare class attributes before `__init__` along with their types.

Reply:
I don't understand what you mean here, if you know how to do it better, please push the changes.

Reply:
Ok, will do.